<launch>
    <node pkg="ntrip_ros" type="ntripclient.py" name="ntrip_client" output="screen">
        <param name="ntrip_server" value="no.nrtk.eu" />
        <param name="ntrip_port" value="9301" />
        <param name="ntrip_user" value="USER" />
        <param name="ntrip_pass" value="PASSWORD" />
        <param name="ntrip_stream" value="MSM_NEAR" />
        <param name="latitude" value="63.416108" />
        <param name="longitude" value="10.401436" />
        <param name="gps_topic" value="/ublox/fix" />
        <param name="rtcm_topic" value="/rtcm" />
    </node>
</launch>
